### PR TITLE
Pull

### PR DESCRIPTION
**Pull request by revanthstrakz**

### CODE REVIEW

**Comment by IamJoker03** [comment]
@revanthstrakz @Adesh15 lolwut

**Comment by krasCGQ** [comment]
Kek @revanthstrakz

**Comment by akhilnarang** [comment]
This made my 3T run 400% faster with quadruple VoLTE

**Comment by krasCGQ** [comment]
Deca VoLTE when

**Comment by Adesh15** [comment]
But this is for mido 🏃🏽‍♂️

**Comment by raphielscape** [comment]
VoLTE+++

Powered by Jio

File: SK/scripts/U [hunk 1]
```diff
@@ -0,0 +1 @@
+uploaded
```
Comment (raphielscape) [inline SK/scripts/U:1]: Boi

**Review by raphielscape — COMMENTED** [review]
For duck sake why Travis

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  - export USE_CCACHE=1
+
+before_script:
+# Download te toolchain
+  - git clone --depth=50 --branch=opt-linaro-7.x https://github.com/krasCGQ/aarch64-linux-android $HOME/LINARO/7.x
+
+# Download the Kernel Sources
+  - git clone --depth=50 --branch=treble https://github.com/revanthstrakz/Stock-Oreo  $HOME/SK
+
+# Download AnyKernel2
```
Comment (krasCGQ) [inline .travis.yml:34]: Someone can explain why

File: SK/scripts/Upload [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (krasCGQ) [inline SK/scripts/Upload:1]: I love this whitespace

**Review by raphielscape — COMMENTED** [review]
> StRaKz-KeRnEl-🐼😑

Oh, trying to following my way to add Emojis on Kernel Version huh? Nice Kang m8

**Pull request closed without merging**